spec/catalog.yaml

```diff
@@ -173,6 +173,33 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/namespaces/{ns}/tables/{table}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+
+    get:
+      tags: [ Namespace ]
+      summary: Get a table from the catalog
+      operationId: GetTable
+      description:
+        Get a table's detailed properties under a specified namespace from the catalog.".
```

Comment:
nit: "detailed information" instead of "detailed properties". also extra `.".` in the end